wgpu-core/src/command/mod.rs

```diff
@@ -64,6 +64,17 @@ impl<B: GfxBackend> CommandBuffer<B> {
         }
     }
 
+    fn get_encoder_read_only(
```

Comment:
Nit: I think it would be better to have this follow thee `get_encoder` `get_encoder_mut` pattern